assets/js/components/data-table.js

```diff
@@ -85,7 +86,7 @@ export const getDataTableFromData = ( data, headers, options ) => {
 							{ showURLs &&
 								<Link
 									className="googlesitekit-table__body-item-url"
-									href={ permaLink }
+									href={ link }
```

Comment:
This should maybe use a full URL here as `link` is a relative path is it not?

Reply:
Let's keep this as `link` for now which is how it originally behaved. We cannot assume anything about `link`, it could be an absolute or relative URL.